src/components/cc-orga-member-list/cc-orga-member-list.js

```diff
@@ -0,0 +1,590 @@
+          : { ...member, state: 'loaded' };
+      }),
+    };
+  }
+
+  /**
+   * The leave event may come from `cc-orga-member-card` or `cc-orga-member-list` itself.
+   * We need to make sure it is caught by the `_checkIsLastAdmin` method that will decide if the event can go through to the smart component.
+   */
+  connectedCallback () {
+    this.addEventListener('cc-orga-member-list:leave', this._checkIsLastAdmin);
+    super.connectedCallback();
+  }
+
+  /**
+   * Everytime we render a new list, remove the "last-admin" errors if the list contains more than 1 admin.
+   */
+  willUpdate (changedProperties) {
+    const isMemberListLoaded = this.members.state === 'loaded' || this.members.state === 'leaving';
+    const updateRelatedToMembers = changedProperties.has('members');
+
+    if (!updateRelatedToMembers || !isMemberListLoaded) {
+      return;
+    }
+
+    const adminList = this._getAdminList();
+
+    if (adminList.length > 1) {
+      this.members = {
+        ...this.members,
+        value: this.members.value.map((member) => ({
+          ...member,
+          error: null,
+        })),
+        lastAdminLeavingError: false,
+      };
+    }
+  }
+
+  render () {
+    const _refTitle = this._memberListHeadingRef;
+    const memberListLoaded = this.members.state === 'loaded' || this.members.state === 'leaving';
+
+    return html`
+
+      <cc-block>
+        <div slot="title">${i18n('cc-orga-member-list.main-heading')}</div>
+        
+        ${this.authorisations.invite ? this._renderInviteForm() : ''}
+        
+        <cc-block-section>
+          <div slot="title" ${ref(_refTitle)} tabindex="-1">
+            ${i18n('cc-orga-member-list.list.heading')}
+            ${this.members.state === 'loaded' ? html`
+              <cc-badge class="member-count" weight="dimmed" intent="neutral" circle>${this.members.value.length}</cc-badge>
+            ` : ''}
+          </div>
+  
+          ${this.members.state === 'loading' ? html`
+            <cc-loader></cc-loader>
+          ` : ''}
+  
+          ${memberListLoaded ? html`
+            ${this._renderMemberList(this.members.value, this.members.identityFilter, this.members.mfaDisabledOnlyFilter)}
+          ` : ''}
+  
+          <!-- TODO replace <cc-error> by <cc-notice> when ready -->
+          ${this.members.state === 'error' ? html`
+            <cc-error>${i18n('cc-orga-member-list.error')}</cc-error>
+          ` : ''}
+        </cc-block-section>
+
+        <cc-block-section>
+          <div slot="title" class="danger">${i18n('cc-orga-member-list.leave.heading')}</div>
+          <div class="leave">
+            <p class="leave__text">
+              ${i18n('cc-orga-member-list.leave.text')}
+            </p>
+            <cc-button
+                danger
+                outlined
+                @cc-button:click=${this._onLeave}
+                ?disabled="${this.members.lastAdminLeavingError}"
+                ?waiting="${this.members.state === 'leaving'}"
```

Comment:
We used to, at the very beginning of this component but we cannot anymore because we use icon only buttons in desktop.
This is also why we decided in the future we would need to add modals :+1:

Reply:
Ho, makes sense! :+1: